packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -781,30 +781,22 @@ export const bookingsRouter = router({
     });
 
     const authorized = async () => {
-      // if the organizer
-      if (booking.userId === user.id) {
-        return true;
-      }
       const eventType = await prisma.eventType.findUnique({
         where: {
           id: booking.eventTypeId || undefined,
+          schedulingType: SchedulingType.COLLECTIVE,
```

Comment:
We have a type error here:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/3504472/232632508-2f67a19b-e4e5-46de-abef-0f2e5a3e0f4f.png">

Reply:
Pushed a fix